R/zchunk_L100.Population_downscale_ctry.R

```diff
@@ -55,23 +165,23 @@ module_socioeconomics_L100.Population_downscale_ctry_DISABLED <- function(comman
     # There's also a `same_precursors_as(x)` you can use
     # If no precursors (very rare) don't call `add_precursor` at all
     tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+      add_title("Population by country, 1700-2010") %>%
+      add_units("thousand") %>%
+      add_comments("Maddison population data cleaned to develop complete data for all years, (dis)aggregated to modern country boundaries") %>%
+      # add_comments("can be multiple lines") %>%
       add_legacy_name("L100.Pop_thous_ctry_Yh") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
+      add_precursors("socioeconomics/socioeconomics_ctry", "Maddison_population") %>%
       # typical flags, but there are others--see `constants.R`
       add_flags(FLAG_NO_TEST, FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
```

Comment:
(This has since been removed from the skeleton-generating script. Not a good idea on my part.)